packages/xo-web/src/xo-app/vm/tab-console.js

```diff
@@ -241,13 +241,18 @@ export default class TabConsole extends Component {
           </Col>
         </Row>
         <Row className='console'>
-          <Col>
-            <NoVnc
+        <Col>
+            {vm.other.disable_pv_vnc === '1' ? (
+              <Container>
+                <p>{_('disabledConsole')}</p>
+              </Container>
```

Comment:
You don't need container if you have only one element.
```suggestion
              _('disabledConsole')
```